pro-planner-client/src/routes/OverviewPage.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import NavigationBar from '../components/NavigationBar';
+import {Card, Col, Container, Row} from 'react-bootstrap';
+import './OverviewPage.scss';
+import ExpenseSplit from '../components/ExpenseSplit';
+import {useSelector} from 'react-redux';
+import Map from '../components/Map';
+import {format, parseISO} from "date-fns";
+
+const OverviewPage = () => {
+    const tripName = useSelector((state) => state.planParameters.name);
+    const location = useSelector((state) => state.planParameters.location);
+    const decidedDates = useSelector(state => state.planParameters.decisionRange);
+    const description = useSelector(state => state.planParameters.description);
+    // const decidedDates = ["2023-07-31T07:00:00.000Z", "2023-08-04T06:59:59.099Z"] // testing purpose
+    // const tripName = 'Trip Name'; // testing purpose
+    // const location = 'Vancouver'; // testing purpose
+    // const description = "description test"; // testing purpose
+
+
+    function processDecidedDates(decidedDates) {
+        if (!decidedDates || decidedDates.length !== 2) {
+            return "Dates are not set"
+        }
+        const [startDate, endDate] = decidedDates;
+        const processedStartDate = format(parseISO(startDate), 'EEE, MMM d yyyy');
+        const processedEndDate = format(parseISO(endDate), 'EEE, MMM d yyyy');
+        return `${processedStartDate} - ${processedEndDate}`
+    }
+
+    const processedDecidedDates = processDecidedDates(decidedDates);
+
+    return (
+        <>
+            <NavigationBar/>
```

Comment:
Another note, if this component is supplied as a child of the home page route (in `index.js`), which it is, that component is placed under a navigation bar, so one doesn't need to be rendered here.